dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/table/model/AnalyticsTableTest.java

```diff
@@ -62,23 +62,23 @@ class AnalyticsTableTest {
   @Test
   void testGetTableNameDataValue() {
     AnalyticsTable table =
-        new AnalyticsTable(AnalyticsTableType.DATA_VALUE, columnsA, Logged.UNLOGGED);
+        new AnalyticsTable(AnalyticsTableType.DATA_VALUE, columnsA, Logged.UNLOGGED, false);
```

Comment:
Perhaps add an enum here for the distribution property, like `Distribution` with values `DISTRIBUTED` and `NONE` or something similar? I know this be a bit overkill, I just think it is nice to use enums as it makes the code instantly more readable, where booleans become hard to read in constructors. Just a suggestoin.